app/workers/mrss_photos_importer.rb

```diff
@@ -63,6 +63,12 @@ def get_attributes(mrss_entry)
   end
 
   def fetch_xml(url)
-    HTTP.headers(user_agent: 'Oasis').timeout(30).follow.get(url).to_s
+    connection = Faraday.new do |faraday|
+      faraday.use(FaradayMiddleware::FollowRedirects)
+      faraday.adapter(:net_http)
+      faraday.headers['User-Agent'] = 'Oasis'
+      faraday.options.timeout = 30
+    end
+    connection.get(url).body
```

Comment:
The original version using the `http` gem is obviously more elegant, but I decided to swap out `http` for `faraday` for a couple of reasons:
- `faraday` is required by the Elasticsearch gems, so it's not going anywhere
- removing `http` reduces the # of gems in our repo, which simplifies gem upgrades in general